scout/server/blueprints/institutes/views.py

```diff
@@ -112,7 +113,7 @@ def lock_filter(institute_id, filter_id):
 def gene_variants(institute_id):
     """Display a list of SNV variants."""
     page = int(request.form.get("page", 1))
-
+    result_size = None
```

Comment:
Maybe remove declaration of `result_size` since it is always set on line 147?

Reply:
result_size is instanced later only of the request is a POST request if it enters in the "else". But if the endpoint gets a GET request "result_size" will be missing and would trigger an error I'm afraid